src/nix_file.rs

```diff
@@ -471,42 +483,77 @@ mod tests {
               ;
               # E
 
-              /**/quuux/**/=/**/5/**/;/*E*/
+              /**/testF/**/=/**/5/**/;/*E*/
 
-              inherit toInherit;
-              inherit (toInherit) toInherit;
+              inherit testG;
+              inherit ({}) testH;
+
+              inherit ({})
+                testI testJ
+                testK testL;
             }
         "#};
 
         std::fs::write(&file, contents)?;
 
         let nix_file = NixFile::new(&file)?;
 
-        // These are `builtins.unsafeGetAttrPos` locations for the attributes
+        // These are `builtins.unsafeGetAttrPos` locations for the attributes, generated by saving
+        // the Nix file above as `test.nix`, then making this `driver.nix` file.
+        //
+        // ```nix
+        // let
+        //   value = import ./test.nix null;
+        //   info =
+        //     name:
+        //     let
+        //       inherit (builtins.unsafeGetAttrPos name value) line column;
+        //     in
+        //     _: ''(${toString line}, ${toString column}, Left("fill me in")),'';
+        // in
+        // builtins.mapAttrs info value
+        // ```
+        //
+        // Then, run this command to get the raw test data. Use any compatible Nix, i.e. Lix.
+        //
+        // ```
+        // nix run nixpkgs#nixVersions.latest -- eval --impure --json --file driver.nix | jq -r .[]
+        // ```
+        //
+        // Fill in the expected data until the test passes.
         let cases = [
-            (2, 3, Right("foo = 1;")),
-            (3, 3, Right(r#""bar" = 2;"#)),
-            (4, 3, Right(r#"${"baz"} = 3;"#)),
-            (5, 3, Right(r#""${"qux"}" = 4;"#)),
-            (8, 3, Right("quux\n  # B\n  =\n  # C\n  5\n  # D\n  ;")),
-            (17, 7, Right("quuux/**/=/**/5/**/;")),
-            (19, 10, Left("inherit toInherit;")),
-            (20, 22, Left("inherit (toInherit) toInherit;")),
+            (2, 3, Right("testA = 1;")),
+            (3, 3, Right("\"testB\" = 2;")),
+            (4, 3, Right("${\"testC\"} = 3;")),
+            (5, 3, Right("\"${\"testD\"}\" = 4;")),
+            (8, 3, Right("testE\n  # B\n  =\n  # C\n  5\n  # D\n  ;")),
+            (17, 7, Right("testF/**/=/**/5/**/;")),
+            (19, 11, Left("testG")),
+            (20, 16, Left("testH")),
+            (23, 5, Left("testI")),
+            (23, 11, Left("testJ")),
+            (24, 5, Left("testK")),
+            (24, 11, Left("testL")),
```

Comment:
These cases need to be run for the locations reported by various Nix versions, such that it works for all of them and not just a single one. Something like

Reply:
With #83 this is now possible. I'll update.